extensions/iam/decentralized-identity/identity-did-service/src/main/java/org/eclipse/dataspaceconnector/identity/DecentralizedIdentityService.java

```diff
@@ -88,7 +87,7 @@ public Result<ClaimToken> verifyJwtToken(TokenRepresentation tokenRepresentation
             PublicKeyWrapper publicKeyWrapper = KeyConverter.toPublicKeyWrapper(publicKeyJwk, publicKey.get().getId());
 
             monitor.debug("Verifying JWT with public key...");
-            Result<Void> verified = VerifiableCredentialFactory.verify(jwt, publicKeyWrapper, audience);
+            Result<Void> verified = JwtUtils.verify(jwt, publicKeyWrapper, audience);
```

Comment:
micro nit: static import?

Reply:
As above - Are you sure? It seems the version without static import is clearer